index.js

```diff
@@ -124,7 +124,7 @@ function makeCachedResolver (prefix, fieldName, cache, originalFieldResolver, po
   })
   return async function (self, arg, ctx, info) {
     if ((skip && await skip(self, arg, ctx, info)) ||
-      (policy && policy.skip && await policy.skip(self, arg, ctx, info))) {
+      (policy && policy.skip && await policy.skip(self, arg, ctx, info)) || info?.operation?.operation === 'mutation') {
```

Comment:
I have a feeling that ? is not available in Node v12.

Anyhow could you move the check before the policy check?

Reply:
You are right. I checked and `?` doesn't work on node 12. I have removed it now and also moved the check before policy check